hera_pspec/pspecdata.py

```diff
@@ -566,16 +566,27 @@ def set_C(self, cov):
         ----------
         cov : dict
             Dictionary containing new covariance values for given datasets and
-            baselines. Keys of the dictionary are tuples, with the first item
-            being the ID (index) of the dataset, and subsequent items being the
-            baseline indices.
+            baselines. Keys of the dictionary are tuples. 
+            The format of a key is like: (dset_idx1, bl1, dset_idx2, bl2, model, conj_1, conj_2, subtracted), 
+            dset_idx1 and bl1 has the same form as the returns of parse_blkey. 
+            `model' should be a str, refers to the name of covariance model. 
+            `conj_1', `conj_2' and `subtracted' should all be boolean.
```

Comment:
@JianrongTan the conj_1 and conj_2 seems like a perhaps unnecessary proliferation of arguments--but please correct me if I am wrong. Is it possible to adopt a convention, say x1 is conj and x2 is non-conj and then force the user to adopt those conventions? In other words, is there any compelling case for why we would want x1 and/or x2 to deviate from that convention?